libs/blocks/global-footer/global-footer.js

```diff
@@ -226,6 +226,12 @@ class Footer {
     if (url.hash !== '') {
       // Hash -> region selector opens a modal
       decorateAutoBlock(regionPickerElem); // add modal-specific attributes
+      if (regionPickerElem.classList[0] !== 'modal') {
```

Comment:
Is this early enough to give us enough context? Or do we also need some logs post `loadBlock(regionPickerElem)` ?

Reply:
What I discussed with Robert is that the issues might be before that call to `loadBlock`, as that [method](https://github.com/adobecom/milo/blob/stage/libs/utils/utils.js#L442) seems to choose the first class. So we need to ensure the first class is not the region picker one